### DISCUSSION TITLE
Changes to dimension stack

### DISCUSSION THREAD

**Discussion by Ookidablobida** [iPortalTeam/ImmersivePortalsMod#496] [Ideas]
It is cool the way it currently it is, but I have some ideas of how you might be able to improve it. The options in world creation would be easier if you could click/drag dimensions or move them up and down sort of like in resourcepack/datapack selecting. Also, I don't know if this is possible, but maybe you could have "inverted" or "connection=up/down or top/bottom" as an option in dimension stack. I thought of this as a way to use dimension stack in conjunction with something like the midnight dimension. You could connect the nether and overworld the same way it normally does, but then when you dig through the bottom of the nether, you dig into the bottom of the midnight. Alternatively, you could just have options for rotation on the x, y and z axes. The y axis could be any rotation, and x and z could be limited to 0 or 180.

**Comment by qouteall** [comment]
It's being planned
